### DISCUSSION TITLE
iptables/post-rules

### DISCUSSION THREAD

**Discussion by xtinct101** [qdm12/gluetun#1400] [Q&A]
Hi,

In the wiki it states to add post iptable rules in the iptables/post-rules.txt file.
Where is this file located or need to be placed??

https://github.com/qdm12/gluetun/wiki/Firewall-options

**Comment by bnhf** [comment]
@xtinct101 

Have you tried a bind mount from the host to the container? So, if you put the file in /data/gluetun/post-rules.txt on the host, and we’re using docker-compose, you’d add this to the volumes section of the Gluetun stack:

```yml
  volumes:
    - /data/gluetun/post-rules.txt:/iptables/post-rules.txt
 ```

Haven’t used this option before myself, but I would think that’s how it's done.

**Reply by damajor** [reply]
Used it and I confirm it works fine ;)
